tech_reports/LLMs/llms.md

```diff
@@ -37,6 +37,17 @@ Authors:
         - [4.10.4.2 Large Matmuls](#41042-large-matmuls)
 
 ## 1. Overview
+What’s the best way to bring up high-performance multi-chip models on Tenstorrent hardware with the Metal stack? We continue to try to answer this question; here are our current best practices, tips and workarounds.
+
+What you need:
+
+* Access to TT hardware. This guide is specifically for bringing models up on wormhole, so whilst most of this advice applies equally to grayskull it is very WH (wormhole)-centric.
```

Comment:
* Access to TT hardware. This guide is specifically for bringing models up on wormhole (WH), so whilst most of this advice applies equally to grayskull it is very WH-centric.